wordpress-mobile/WordPress-Aztec-Android
# Issue/280 update heading icon

### Fix
Update heading icon to use [`gridicons-heading`](http://automattic.github.io/gridicons/) as described in https://github.com/wordpress-mobile/WordPress-Aztec-Android/issues/280.

### Test
1. Open demonstration app.
2. Notice ***Heading*** format button icon default state.
3. Long-press ***Heading*** format button.
4. Notice ***Heading*** format button icon pressed state.
5. Tap ***HTML*** format button.
6. Notice ***Heading*** format button icon disabled state.

### Review
@mattmiklic to review new icon.
